aws/resource_aws_msk_scram_secret.go

```diff
@@ -0,0 +1,216 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kafka"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAwsMskScramSecret() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsMskScramSecretCreate,
+		Read:   resourceAwsMskScramSecretRead,
+		Update: resourceAwsMskScramSecretUpdate,
+		Delete: resourceAwsMskScramSecretDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"cluster_arn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"secret_arn_list": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
```

Comment:
similar nit as above to provide Arn validation for each item in the list/set e.g.
```suggestion
				Elem:     &schema.Schema{
				    Type: schema.TypeString,
				    ValidateFunc: validateArn,
			         },
```